container-chains/templates/frontier/runtime/Cargo.toml

```diff
@@ -30,7 +30,10 @@ frame-support = { workspace = true }
 frame-system = { workspace = true }
 frame-system-rpc-runtime-api = { workspace = true }
 frame-try-runtime = { workspace = true, optional = true }
-pallet-balances = { workspace = true }
+# TODO: we enable this feature here, but this feature should not be enabled when compiling
+# the other substrate-based runtimes. Check if cargo does enabled it automatically because
+# it probably does (otherwise it would have to compile this crate twice)
+pallet-balances = { workspace = true, features = ["insecure_zero_ed"] }
```

Comment:
I verified this, and it depends on how we run `cargo build`. With our current CI setup, both templates are built at the same time using one cargo command, so both templates have this feature enabled.

Building them separately like this, the feature is only enabled in the frontier template:

```
cargo build --release --features=runtime-benchmarks -p container-chain-template-simple-node
cargo build --release --features=runtime-benchmarks -p container-chain-template-frontier-node
```

But I don't think this is an issue, because the only differences when compiling with this feature are:
* Disable an assert that the existential deposit is not zero
* Change the impl of have_providers_or_no_zero_ed from always returning true to doing some computation (storage read?)

```rust
        /// Returns `true` when `who` has some providers or `insecure_zero_ed` feature is disnabled.
        /// Returns `false` otherwise.
        #[cfg(not(feature = "insecure_zero_ed"))]
        fn have_providers_or_no_zero_ed(_: &T::AccountId) -> bool {
            true
        }

        /// Returns `true` when `who` has some providers or `insecure_zero_ed` feature is disnabled.
        /// Returns `false` otherwise.
        #[cfg(feature = "insecure_zero_ed")]
        fn have_providers_or_no_zero_ed(who: &T::AccountId) -> bool {
            frame_system::Pallet::<T>::providers(who) > 0
        }
```